### DISCUSSION TITLE
How are you using VPS?

### DISCUSSION THREAD

**Discussion by uasdj25** [htpcBeginner/docker-traefik#223] [Q&A] [answered]
Hi all,

im completely new to this topic.  I read in a couple of articles that Anand leverages one. Just curious if it’s to layer security, leverage cloud resources?  Curious as to how I might fit this into my setup.  If anyone has pointers on where to learn about it, I’d appreciate it.

**Comment by Maynardnaze** [comment, marked as answer]
I use a mix of his web and regular docker-conpose.yml 

My advice is get a traefik working then add container one by one. Follow his guide exactly and you'll be fine.
